.github/codeql-config.yaml

```diff
@@ -0,0 +1,12 @@
+paths-ignore:
```

Comment:
Please run yamllint the all the .yaml on this file and fix the missing new lines at the end.

Reply:
Only issue left isn't a valid error for this file:

```
yamllint .github/codeql-config.yaml
.github/codeql-config.yaml
  2:13      error    syntax error: mapping values are not allowed here (syntax)
```